--- original
+++ derived
@@ -42,7 +42,7 @@
 
 I hold my weapons ready and step close enough to make him focus on me.
 
-“Gog, yes, yes. Hey, Gog, listen. We did not intend to hurt you or your boss, but we had to help our friend Hamun Kost. Like, and also protect the land from all the, you know, threats and the robberies that you guys have been doing. I just wanted to know before, you know, we finished the business, like, why are we doing, and do you know anything about Hamun Kost? Tell us what has happened in the past.”
+“Gog, yes, yes. Hey, Gog, listen. We did not intend to hurt you or your boss, but we had to help our friend Hamun Kost. Like, and also protect the land from all the, you know, threats and the robberies that you guys have been doing. I just wanted to know before, you know, we finished the business, like, why are you doing this, and do you know anything about Hamun Kost? Tell us what has happened in the past.”
 
 Gog looks at me.
 
