--- original
+++ derived
@@ -42,7 +42,7 @@
 
 Four bugbears drove us out before. This cave has four bugbears, an ogre, and the orc who has somehow made all of them listen.
 
-“Well, you make a point, but that's pretty valid. So, what do you suggest? Do you suggest to kill the orc from this far, or do you wanna confront him? Because if we confront him, like, he's gonna… how about if he alerts the bugbears and the ogre?”
+“Well, you make a point—that's pretty valid. So, what do you suggest? Do you suggest to kill the orc from this far, or do you wanna confront him? Because if we confront him, like, he's gonna… how about if he alerts the bugbears and the ogre?”
 
 Sister Maela looks carefully past us into the cave.
 
